python/pyarrow/tests/test_dataset.py

```diff
@@ -3130,6 +3130,58 @@ def test_csv_fragment_options(tempdir, dataset_reader):
         pa.table({'col0': pa.array(['foo', 'spam', 'MYNULL'])}))
 
 
+def test_encoding(tempdir, dataset_reader):
+    path = str(tempdir / 'test.csv')
+
+    for encoding, input_rows, expected_table in [
```

Comment:
`expected_table` here is unused.

Reply:
Thanks for catching that, it's been removed in [b3ac697](https://github.com/apache/arrow/pull/13820/commits/b3ac6979b0e78c96401dfe03298003287d1f9b7e)